docs/source/topics/backends/snmp.rst

```diff
@@ -0,0 +1,149 @@
+
+************
+SNMP backend
+************
+
+SNMP
+====
+
+SNMP (Simple Network Management Protocol) is a network protocol very useful for retrieving info from a device.
+All the information is retrieved by using codes called MIBs. All MIBs have a tree-like structure, every main information is the root and by adding more detail to the info
+the tree gains more depth.
+Obviously, by getting the smallest MIB which is "1" or simply " . " one can get all the tree.
+
+The base SNMP backend contains the following methods (some internal methods are not documented and are subject to change in the future):
```

Comment:
I think we can avoid explaining what SNMP is, we assume peopel using this library will know.

We can however include information about how all the data is retrieved and give a few hints on how to use the base backend to create new SNMP backends for other systems (eg: indicate the python path for the base backend and explain anything important regarding it).